packages/gatsby-source-contentful/src/normalize.js

```diff
@@ -49,24 +49,34 @@ const fixId = id => {
 }
 exports.fixId = fixId
 
-const fixIds = object =>
-  _.mapValues(object, (val, key) => {
-    if (key === `sys`) {
-      val = {
-        ...val,
-        id: fixId(val.id),
-        contentful_id: val.id,
-      }
-    }
+const fixIds = object => _fixIds(object)
 
-    if (_.isArray(val)) {
-      return _.toArray(fixIds(val))
-    }
-    if (_.isPlainObject(val)) {
-      return fixIds(val)
+const _fixIds = (object, front = new Set()) => {
+  if (!object || typeof object !== `object`) {
+    return undefined
```

Comment:
This should be the same as undefined. I'm okay to keep undefined if you want to make it explicit
```suggestion
    return
```

Reply:
eslint says no

Reply:
Btw the lint rule is bs. I hate it. But that's the only reason I'm doing this.

Reply:
I double checked it. In a plot twist it's now allowing me to omit it.
I think it was complaining about the `return object.forEach` and then requiring all other returns to be explicit. I still do not like this lint rule.